### PR TITLE
SMS Languages Page Error

### PR DESCRIPTION
**Pull request by pnaw94**

### CODE REVIEW

**Comment by nickpell** [comment]
what error is this PR trying to fix? It looks related to the same page 500ing: http://manage.dimagi.com/default.asp?217054

File: corehq/apps/style/templates/style/includes/bootstrap3/ko.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {% load hq_shared_tags %}
-<script type="text/javascript" src="{% new_static 'knockout/dist/knockout.js' %}"></script>
+{% if request.upgrade_knockout_js %}
```
Comment (nickpell) [inline corehq/apps/style/templates/style/includes/bootstrap3/ko.html]: It looks like this if-else should be reversed

**Comment by nickpell** [comment]
@biyeun can you take a look at this PR?

**Comment by biyeun** [comment]
offlining

**Comment by gcapalbo** [comment]
@nickpell sounds like this might have already been taken care of, but the original context for this PR was: http://manage.dimagi.com/default.asp?216371

**Pull request merged**